### ISSUE TITLE
Adding Vietnamese

### ISSUE THREAD

**Issue by crystaltype** [daltonmaag/ubuntu#67]
Hi,

I see that this type doesn't have Vietnamese yet.

I'm willing to contribute the complete Vietnamese character set directly, by sending you a 'vietnamese only' glyphapp file for you to review and integrate (Or make changes myself and make a pull request.)

Or, you can make changes and then push your work to master and I will give further feedback :)

Please respond ASAP. Thanks!

P/S: I'm a Vietnamese native speaker and a self-study graphic designer/type designer. I have been working with fonts for over 5 years! I'm here to (hopefully) make google fonts a better place for people using Vietnamese! :) Thanks!

cc': @davelab6

**Comment by moyogo** [comment]
Hi @crystaltype. Great!

What glyphs do you plan on adding?
Are you adding the combining marks (acutecomb, gravecomb, circumflexcomb, tildecomb, brevecomb, hookabovecomb, horncomb, dotbelowcomb) or will you be using the spacing ones already in the fonts (acute, grave, circumflex, tilde, breve, dotaccent) and only add the ones not existing in any form (hookabovecomb, horncomb)? The combining ones would be better but they are not there.
Are you going to build the stacking diacritics from those or will you be adding tighter combined ones as single glyphs (circumflex_acute, etc. like dieresis_acute)? Combined ones as single glyphs would allow for better design unless you’re happy with composing with existing ones.

The glyphsLib library should soon allow to go back and forth between UFOs and Glyphs. You could use the development version if you want.

**Comment by crystaltype** [comment]
Yes, I'll be adding the combining marks (but are composed from your already existed marks), because Vietnamese does use some of the other languages characters, so I won't be changing those (such as aacute, agrave etc...). I'll add hookabovecomb, horncomb and dotbelowcomb. Then I'll add their stacking diacritics, which will be used for Vietnamese, for example (circumflexcomb_acutecomb, brevecomb_acutecomb etc...).

Thank you, I'll begin working on them as soon as possible. Do you want me to send you an email containing the glyphs file or I can just fork the repo and work from that?

**Comment by madig** [comment]
If you want to use Glyphs, I hope to have improved ufo2glyphs and glyphs2ufo scripts soon. You can just fork the repo and we can turn the Glyphs file into UFOs later.
